data-serving/scripts/converters.py

```diff
@@ -115,3 +116,54 @@ def convert_demographics(id: str, age: str, sex: str) -> Dict[str, Any]:
 
     # If the dictionary is empty, we want to omit it from the JSON output
     return demographics if demographics else None
+
+
+def convert_location(id: str, location_id: float, country: str, adminL1: str, adminL2: str, locality: str, latitude: float, longitude: float) -> Dict[str, Any]:
```

Comment:
Can we linebreak this signature?

Reply:
got this largely to work automatically with

```
  "python.linting.pep8Args": [
    "--max-line-length=100"
  ],
  "python.formatting.autopep8Args": [
    "--max-line-length", "100", "--experimental"
  ],

```
though it still can't break long comments -- but i got a sweet vertical ruler in vscode: `"editor.rulers": [100]` -- so I can see when it's a problem!